cpp/src/groupby/sort/group_single_pass_reduction_util.cuh

```diff
@@ -65,7 +65,7 @@ struct reduce_functor {
     using OpType     = cudf::detail::corresponding_operator_t<K>;
     using ResultType = cudf::detail::target_type_t<T, K>;
 
-    auto result_type = is_fixed_point<T>()
+    auto result_type = is_fixed_point<ResultType>()
```

Comment:
I understand the error and why it is being thrown, however, I don't understand how we get to this point. `T` is `fixed_point` and `ResultType` is not? Potentially this is for the `ARGMIN` or `ARGMAX` case above? 

Regardless, the change looks good.

Reply:
Yes. The test is for `ARGMIN` which has a hardcoded target-type of `size_type` regardless of source-type.
https://github.com/rapidsai/cudf/blob/cf8c73aed29179b3165a92a10faac6ca45ccd6d0/cpp/include/cudf/detail/aggregation/aggregation.hpp#L527-L531
Same for `ARGMAX`.